### PR TITLE
Add BaseConnectorTest test to verify failure when creating tables in a schema where it doesn't exist

### LINKED ISSUE #11794: Add BaseConnectorTest test to verify failure when creating tables in a schema where it doesn't exist
Relates to #11110

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Add BaseConnectorTest test to verify failure when creating tables in a schema where it doesn't exist. Also, add support for creating and dropping schemas in Accumulo connector.
Fixes #11794

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
(x) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by ebyhr** [comment]
CI hit #11764

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java [hunk 1]
```diff
@@ -238,6 +238,14 @@ public void testCreateTableAsSelect()
                 .hasStackTraceContaining("This connector does not support creating tables with data");
     }
 
+    @Test
+    @Override
+    public void testCreateTableAsSelectSchemaNotFound()
+    {
+        assertThatThrownBy(super::testCreateTableAsSelectSchemaNotFound)
+                .hasStackTraceContaining("This connector does not support creating tables with data");
```
Comment (findepi) [inline plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java:246]: remove the test override, this should be handed by the check of the conenctor behavior

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java [hunk 2]
```diff
@@ -1940,6 +1958,24 @@ public void testCreateTableAsSelect()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateTableAsSelectSchemaNotFound()
+    {
+        // Covered by testCreateTableAsSelect
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE));
```
Comment (findepi) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:1965]: `SUPPORTS_CREATE_TABLE_WITH_DATA`

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java [hunk 3]
```diff
@@ -1868,6 +1868,24 @@ public void testCreateTable()
         assertFalse(getQueryRunner().tableExists(getSession(), tableNameLike));
     }
 
+    @Test
+    public void testCreateTableSchemaNotFound()
+    {
+        // Covered by testCreateTable
```
Comment (findepi) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:1874]: remove (it's kind of obvious this is not "the test" for create table)

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java [hunk 4]
```diff
@@ -1940,6 +1958,24 @@ public void testCreateTableAsSelect()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateTableAsSelectSchemaNotFound()
+    {
+        // Covered by testCreateTableAsSelect
```
Comment (findepi) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java:1964]: ok to remove

File: plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java [hunk 5]
```diff
@@ -275,6 +275,22 @@ public void testCreateTable()
         throw new SkipException("TODO");
     }
 
+    @Test
+    @Override
```
Comment (findepi) [inline plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java:279]: document why override?

**Review by findepi — APPROVED** [review]

**Review by hashhar — APPROVED** [review]

**Pull request merged**